### PR TITLE
Support Team CC Docker Image Publishing

### PR DESCRIPTION
**Pull request by msnwatson**
## What was the problem?

Prior to this PR, automated scripting for defining variables for docker images did not support docker files in subdirectories.

Associated tickets or Slack threads:

## How does this fix it?[^1]

Alters the `image-name.sh` script to strip out slashes in the bash variable names and to replace slashes with dashes in the docker image names

## How to test this PR
- New script has been tested by running it and getting the new output file

[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).

### CODE REVIEW

**Review by yoomlam — COMMENTED** [review]
I don't think this PR is needed.
For example, see line 10, where the gradle folder can be in any subdirectory.

File: scripts/image-names.sh [hunk 1]
```diff
@@ -14,21 +14,22 @@ gradle_folder() {
...
 
 # These names must match the images specified in Helm configs
 prod_image_name() {
-  echo "vro-$1"
+  echo "vro-${1//[\/]/-}"
 }
 
 # These names should match directory names
-IMAGES=( api-gateway app postgres db-init console svc-bgs-api svc-lighthouse-api )
+IMAGES=( api-gateway app postgres db-init console svc-bgs-api svc-lighthouse-api domain-cc/svc-cc-app )
```
[the remark this answers is not in the document]
Comment (msnwatson) [reply scripts/image-names.sh:32]: i ran the script w/o my changes and with this change and i got error

```
=== Verifying folders and files
ls: ./svc-cc-app/build.gradle: No such file or directory
```

[the remark this answers is not in the document]
Comment (yoomlam) [reply scripts/image-names.sh:32]: After line 10 in `image-names.sh`, add something like:
```
svc-cc-app) echo "domain-cc/$1";;
```

File: scripts/image-names.sh [hunk 2]
```diff
@@ -18,6 +19,7 @@ gradle_image_name() {
 }
 
 # Bash variables can't have dashes, so strip them out of the directory names
+# also strip out subdirectory slashes
```
Comment (yoomlam) [inline scripts/image-names.sh]:
```suggestion
```

**Review by yoomlam — APPROVED** [review]

File: scripts/image-names.sh [hunk 3]
```diff
@@ -27,8 +28,9 @@ prod_image_name() {
   echo "vro-$1"
 }
 
-# These names should match directory names
-IMAGES=( api-gateway app postgres db-init console svc-bgs-api svc-lighthouse-api )
+# These names should match directory names, if the docker image is built in a
+# subdirectory, be sure to add the sub directory to the gradle_folder function above
```
Comment (yoomlam) [inline scripts/image-names.sh:32]: Excellent!

**Comment by lukey-luke** [comment]
Thanks @msnwatson for your support!

**Pull request merged**